app/src/main/java/com/example/plantsapp/presentation/ui/profile/ProfileFragment.kt

```diff
@@ -51,8 +54,40 @@ class ProfileFragment : Fragment(R.layout.fragment_profile) {
             )
 
             tvProfileName.text = user.name
+        }
+    }
+
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        inflater.inflate(R.menu.menu_profile_appbar, menu)
+        with(viewModel) {
+            isSignOutBtnVisible.observe(viewLifecycleOwner) {
+                menu.findItem(R.id.action_sign_out)
+                    .isVisible = it
+            }
+        }
```

Comment:
It is not necessary to always use scope function with viewmodel
you can just write 
viewModel.isSignOutBtnBisible.observe.....